quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/storage/infinispan/jgroups/JGroupsConfigurator.java

```diff
@@ -107,7 +90,6 @@ public static JGroupsConfigurator create(ConfigurationBuilderHolder holder) {
         var configurator = new ArrayList<JGroupsStackConfigurator>(3);
```

Comment:
This may be set back to 2 based on the diff from commit: 

https://github.com/keycloak/keycloak/commit/55998367d899b7eda9fb97752b241ed69240cdba#diff-b928b666e8e3c4cba204462c558ee57541362ac9b9fc5a08135ec33358a2cd6dR107